tests/rptest/tests/metrics_test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.services.admin import Admin
+from rptest.services.cluster import cluster
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.types import TopicSpec
+
+BOOTSTRAP_CONFIG = {
+    'aggregate_metrics': False,
+    'disable_metrics': False,
+}
+
+
+class MetricsTest(RedpandaTest):
+    def __init__(self, *args, **kwargs):
+        rp_conf = BOOTSTRAP_CONFIG.copy()
+
+        super(MetricsTest, self).__init__(*args,
+                                          extra_rp_conf=rp_conf,
+                                          **kwargs)
+
+        self.admin = Admin(self.redpanda)
+
+    @cluster(num_nodes=3)
+    def test_aggregate_metrics(self):
```

Comment:
Since the code path to have metrics aggregated initially is a different one, can you add a similar test for that?

Reply:
Change to parameterized test to test both initial states.

Reply:
Thanks, looks good!